src/class.ziprecipes.php

```diff
@@ -1454,6 +1468,32 @@ public static function zrdn_insert_db($post_info) {
         return $recipe_id;
     }
 
+    public static function zrdn_calculate_total_time($post_info) {
+        $total_time = NULL;
+        $prep_time_hours = (!empty(Util::get_array_value("prep_time_hours", $post_info))) ? Util::get_array_value("prep_time_hours", $post_info) : NULL;
+        $prep_time_minutes = (!empty(Util::get_array_value("prep_time_minutes", $post_info))) ? Util::get_array_value("prep_time_minutes", $post_info) : NULL;
+        $cook_time_hours = (!empty(Util::get_array_value("cook_time_hours", $post_info))) ? Util::get_array_value("cook_time_hours", $post_info) : NULL;
+        $cook_time_minutes = (!empty(Util::get_array_value("cook_time_minutes", $post_info))) ? Util::get_array_value("cook_time_minutes", $post_info) : NULL;
+        if ($prep_time_hours || $prep_time_minutes || $cook_time_hours || $cook_time_minutes) {
+            $prep_time_total = sprintf("%02d", $prep_time_hours) . ':' . sprintf("%02d", $prep_time_minutes) . ':00';
+            $cook_time_total = sprintf("%02d", $cook_time_hours) . ':' . sprintf("%02d", $cook_time_minutes) . ':00';
+            $secs = strtotime($cook_time_total) - strtotime("00:00:00");
+            $total_time = date("H:i:s", strtotime($prep_time_total) + $secs);
```

Comment:
But we are not using seconds like PT1H15M

Reply:
Why is it called `$secs`?

Reply:
just trying for plural but i forgot sec are plural as well 
I will correct

Reply:
Issues isn't plural but since you're saying "we are not using seconds", the value has nothing to do with second(s) at all, right? So why is `$secs` needed at all?